cmd/cli/main.go

```diff
@@ -168,6 +168,33 @@ func UpdateAction(c *cli.Context) error {
 	return nil
 }
 
+// TerminateAction executes the CLI terminate command
+func TerminateAction(c *cli.Context) error {
+	terminate := operations.TerminateJob{}
+
+	jobNameBase := c.String("job-name-base")
+	if len(jobNameBase) != 0 {
```

Comment:
With the mode you only have an if clause and after the if you assing `terminate.Mode = mode`. May be neater to do the same with this argument:

```
jobNameBase := c.String("job-name-base")
if len(jobNameBase) == 0 {
  return cli.NewExitError("unspecified flag 'job-name-base'", -1)
}
terminate.JobNameBase = jobNameBase
```